testsuite/tests/singlecluster/mtls/test_mtls_configuration.py

```diff
@@ -0,0 +1,86 @@
+"""
+Tests mTLS configuration and readiness after Kuadrant CR changes
+"""
+
+import time
+import pytest
+
+from openshift_client import selector
+from testsuite.tests.singlecluster.mtls.conftest import wait_for_mtls_status
+
+pytestmark = [pytest.mark.kuadrant_only]
+
+
+def test_peer_authentication_resource_applied(kuadrant, cluster, rate_limit):  # pylint: disable=unused-argument
+    """Verify PeerAuthentication with STRICT mode is created when mTLS is enabled"""
+    kuadrant.refresh()
+    kuadrant.model.spec.mtls.enable = True
```

Comment:
You dont need to do it, but know that `.refresh()` returns the object thats been called on so you can use this nice factory pattern and put two lines into one :smile: 
`kuadrant.refresh().model.spec.mtls.enable = True`